packages/multichain-network-controller/src/utils.ts

```diff
@@ -33,3 +37,43 @@ export function checkIfSupportedCaipChainId(
   // Check if the chain id is supported
   return Object.keys(AVAILABLE_MULTICHAIN_NETWORK_CONFIGURATIONS).includes(id);
 }
+
+/**
+ * Updates a network configuration to the format used by the MultichainNetworkController.
+ * This method is exclusive for EVM networks with hex identifiers from the NetworkController.
+ *
+ * @param network - The network configuration to update.
+ * @returns The updated network configuration.
+ */
+export const updateNetworkConfiguration = (
+  network: NetworkConfiguration,
+): MultichainNetworkConfiguration => {
+  return {
+    chainId: toCaipChainId('eip155', hexToNumber(network.chainId).toString()),
```

Comment:
Would be better to use the `KnownCaipNamespace` enum we have on `@metamask/utils` IMO.

Reply:
done in https://github.com/MetaMask/core/pull/5315/commits/59868b4fc9e19434e04ec8b8dcf394c33468c06c